MFVNeutralino/plugins/Vertexer.cc

```diff
@@ -366,7 +366,7 @@ MFVVertexer::MFVVertexer(const edm::ParameterSet& cfg)
       hs_output_vertex_ntracks[step] = fs->make<TH1F>("h_output_"+stepStrs[step]+"_vertex_ntracks", ";ntracks/vtx", 30, 0, 30);
       hs_output_vertex_mass[step] = fs->make<TH1F>("h_output_"+stepStrs[step]+"_vertex_mass", ";mass/vtx (GeV)", 20, 0, 1000);
       hs_output_vertex_track_weights[step] = fs->make<TH1F>("h_output_"+stepStrs[step]+"_vertex_track_weights", ";vertex track weights", 21, 0, 1.05);
-      hs_output_vertex_chi2[step] = fs->make<TH1F>("h_output_"+stepStrs[step]+"_vertex_chi2", ";normalized chi2", 20, 0, max_seed_vertex_chi2);
+      hs_output_vertex_chi2[step] = fs->make<TH1F>("h_output_"+stepStrs[step]+"_vertex_chi2", ";normalized chi2", 10, 0, 10);

```

Comment:
It's probably fine, but I thought the finer granularity chi2 plots were actually useful? Since 10 bins from 0 to 10 means most of the interesting behavior we want to look at is all in the first or second bin.

Reply:
Okay, I have changed it to be finer by 4 times. Done.